Backend/Models/ApiContext.cs

```diff
@@ -15,7 +15,7 @@ public class ApiContext : DbContext
         public DbSet<DayOfWeek> DaysOfWeek { get; set; }
         public DbSet<WorkingHours> WorkingHours { get; set; }
         public DbSet<PharmacyWorkingHours> PharmacyWorkingHours { get; set; }
-        public DbSet<WorkerState> WorkerState { get; set; }
+        public DbSet<EmployeeState> WorkerState { get; set; }
```

Comment:
Shouldn't variable name be change as well?

Reply:
Fixed